runtime/plugin/health.vim

```diff
@@ -1 +1,7 @@
-command! -nargs=* CheckHealth call health#check([<f-args>])
+function! s:complete(lead, _line, _pos) abort
+  return filter(map(glob($VIMRUNTIME.'/autoload/health/*', 1, 1),
```

Comment:
Oh, wow. `:h globpath()` even mentions this exact example. Using `&rtp` instead of `$VIMRUNTIME` is even better, of course.

Thanks for the pointer!